internal/api/handlers.go

```diff
@@ -356,3 +364,16 @@ func logErrorAndWriteResponse(msg string, status int, w http.ResponseWriter) {
 		logErrorAndWriteResponse(err.Error(), http.StatusInternalServerError, w)
 	}
 }
+
+var GeneratePassword = func(length int) (string, error) {
```

Comment:
We could instead of returning a (string, error), return just a string and return `""` when an error occurs. This is kinda subjective.